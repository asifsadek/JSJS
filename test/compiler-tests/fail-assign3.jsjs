val x : num = 10;
val y : bool = true;
val s : string = "hello";

val z = x + y; // Fail: adding incompatible types

