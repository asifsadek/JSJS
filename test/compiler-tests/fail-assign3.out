Type error: Cannot have types 'num' and 'bool' for operator '+'
